pkg/shares/shares.go

```diff
@@ -25,7 +25,7 @@ func (n NamespacedShare) Data() []byte {
 }
 
 // NamespacedShares is just a list of NamespacedShare elements.
-// It can be used to extract the raw raw shares.
+// It can be used to extract the raw shares.
 type NamespacedShares []NamespacedShare
```

Comment:
tbh, we should get rid of this struct as its pointless middleware. This is remnant of the first implementation done two years ago

Reply:
Sounds good! I can tackle after we merge this PR and https://github.com/celestiaorg/celestia-app/pull/691